staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -145,6 +145,9 @@ type configController struct {
 	// to a given FlowSchema in any minute.
 	// This may only be accessed from the one and only worker goroutine.
 	mostRecentUpdates []updateAttempt
+
+	// watchTracker implements the necessary WatchTracker interface.
+	WatchTracker
```

Comment:
`NewWidthEstimator` would need two interfaces to be initialized:
- `utilflowcontrol.WatchTracker` and 
- `StorageObjectCountTracker`

if we add a field `WatchTracker` to `server.Config` then we can feed both interfaces to `NewWidthEstimator`

Reply:
You can take a look at https://github.com/kubernetes/kubernetes/pull/103539/commits/1ddb305481a8f2ab74883494f7448e2906bb5641 what I've done